.github/workflows/ci.yml

```diff
@@ -83,6 +83,14 @@ jobs:
 
     - name: Build Container Image
       run: |
-        docker buildx build --platform linux/amd64 --platform linux/arm64 \
-        -t build-container-test .
+        docker buildx build --platform linux/amd64 --load \
+        -t build-container-test:latest .
+
+    - name: Test Container Image
+      run: |
+        docker run --rm \
+          -e VET_API_KEY=${{ secrets.SAFEDEP_CLOUD_API_KEY }} \
```

Comment:
@KunalSin9h `secrets` are not available when a PR is raised from forked repository due to security reason. So this job will always fail when someone from the community raises a PR from forked repo.

What I suggest is you run the `Test Container Image` step only if the PR is from the same repository and not forked repository. See example:

https://github.com/safedep/vet/blob/main/.github/workflows/vet-ci.yml#L33